src/stringutils.cpp

```diff
@@ -154,6 +165,29 @@ std::optional<int64_t> stoi64(const std::string& str) {
     }
 }
 
+// TODO: Write unit test
+std::optional<float> stof(const std::string& str) {
```

Comment:
Add tests

Reply:
done, and added tests for new stou64 as well (size_t)